runtime/ftplugin/help.vim

```diff
@@ -20,3 +20,6 @@ endif
 
 let &cpo = s:cpo_save
 unlet s:cpo_save
+
+" Open the help file when pressing 'K'.
+nnoremap <silent><buffer> K :exe 'help '.HelpTopic()<CR>
```

Comment:
This is better expressed as

    :execute 'help' help#topic()<CR>

(note: no space after `help` in a string, no concat operator).